dpnp/dparray.pyx

```diff
@@ -47,6 +47,47 @@ cimport numpy
 cimport dpnp.dpnp_utils as utils
 
 
+# initially copyed from original
```

Comment:
copyed->copied

Reply:
done

Reply:
@Alexander-Makaryev it looks like it is related to #89